src/lib/storage/snapshots_finish.rb

```diff
@@ -0,0 +1,48 @@
+require "yast2/fs_snapshot"
+require "installation/finish_client"
+
+module Storage
+  class SnapshotsFinish < ::Installation::FinishClient
+    include Yast::I18n
+
+    def initialize
+      textdomain "storage"
+
+      Yast.import "Mode"
+      Yast.import "StorageSnapper"
+      Yast.include self, "installation/misc.rb"
```

Comment:
Yes, I agree, I prefer that class living on the Yast namespace. But I took a look at this and I though it was some kind of agreement: https://github.com/yast/yast-bootloader/blob/master/src/lib/bootloader/finish_client.rb